### PR TITLE
text edits and citations

### PR DESCRIPTION
**Pull request by collnell**
Changes made:
-----------
Added citations for now. Cleaned up the text in the Intro, SNOTEL map, and ridgeline sections. Let's either add the actual links or remove any links that don't direct to the correct page before we send out. 

Can you take care of the same tasks for the sections you are the point person for? I added a lot of text feedback, but ultimately it comes down to how you decide to deliver those parts.

Testing:
----------------------------
Before making this pull request, I:
- [ ] Cleaned the code the way Vue likes it - run 'npm run lint --fix'        
- [ ] Made sure all tests run
- [ ] Ran WAVE plugin 508 compliance tool

I can confirm this has been checked on:
- [ ] Chrome
- [ ] Safari
- [ ] Edge
- [ ] Firefox
- [ ] Samsung Internet
- [ ] Internet Explorer 11 (not supported, but still needs at least a working user redirect page)

### CODE REVIEW

File: src/assets/text/referencesText.js [hunk 1]
```diff
@@ -3,14 +3,68 @@ export default {
...
+        reference:
+          "14. McCabe, G. J. & Clark, M. P. Trends and Variability in Snowmelt Runoff in the Western United States. J. Hydrometeorol. 6, 476–482 (2005).",
+      },
+      {
+        reference:
+          "15. Dettinger, M. D. & Cayan, D. R. Large-Scale Atmospheric Forcing of Recent Trends toward Early Snowmelt Runoff in California. J. Clim. 8, 606–623 (1995).",
+      },
+      {
+        reference:
+          "16. López-Moreno, J. I. et al. Snowpack variability across various spatio-temporal resolutions. Hydrol. Process. 29, 1213–1224 (2015).",
```
Comment (ellenbechtel) [inline src/assets/text/referencesText.js:67]: omg THANK YOU for doing this

File: src/components/Intro.vue [hunk 2]
```diff
@@ -8,47 +8,18 @@
     <!-- TAKEAWAY TITLE -->
     <template v-slot:takeAway>
       <h2>
-        A <a
-          href="https://agupubs.onlinelibrary.wiley.com/doi/full/10.1002/2017GL073551"
-          target="_blank"
-        >majority</a> of the water in the western U.S. comes from snowmelt.
+        A majority of the water in the western U.S. comes from snowmelt<sup>1</sup>.
```
Comment (ellenbechtel) [inline src/components/Intro.vue:11]: what's <sup>?

Comment (ellenbechtel) [reply src/components/Intro.vue:11]: OH MY GOD COOL ^ it just made the question mark a superscript, I had no idea that that was an HTML tag!

**Review by ellenbechtel — APPROVED** [review]
awesome

File: src/components/Intro.vue (hunk 2 above)
Comment (collnell) [reply src/components/Intro.vue:11]: lolololol. I did that for citations. It's not the most glamorous way to do it, but it's legit. We can always style a little better clsoer to release

**Pull request merged**